e2e-product-testing/cypress/integration/AdjustPlanningUnits.feature

```diff
@@ -0,0 +1,63 @@
+Feature: 'Exclude' planning units
```

Comment:
pro-tip for next time: When all the scenarios of a Feature shares the same 'pre-condition' step, you can use:

`Feature: xxxxx`
   Background: Given the user is in XXXX
      Scenario: AAA
      Sceario: BBB